### ISSUE TITLE
Support for GHC 8.4.1

### ISSUE THREAD

**Issue by varosi** [commercialhaskell/intero#539]
Currently with the latest nightly it give this error:
```
src\GhciMonad.hs:48:1: error:
        Could not find module `GHCi.ObjLink'
        Use -v to see a list of the files searched for.
       |
    48 | import GHCi.ObjLink as ObjLink
       | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by chrisdone** [comment]
What version of intero?

**Comment by varosi** [comment]
Building against nightly-2018-04-10 on Windows x64 of 37af01509e55341ffcc81b56d1f30032a055d1e5 still give that error.

**Comment by varosi** [comment]
> Downloaded nightly-2018-04-10 build plan.
> Populated index cache.
> syb-0.7: configure
> network-2.6.3.5: download
> syb-0.7: build
> Win32-2.5.4.1: configure
> Win32-2.5.4.1: build
> network-2.6.3.5: configure
> syb-0.7: copy/register
> network-2.6.3.5: build
> network-2.6.3.5: copy/register
> Win32-2.5.4.1: copy/register
> time-1.8.0.2: configure
> time-1.8.0.2: build
> time-1.8.0.2: copy/register
> random-1.1: configure
> directory-1.3.1.5: configure
> random-1.1: build
> random-1.1: copy/register
> directory-1.3.1.5: build
> directory-1.3.1.5: copy/register
> ghc-boot-8.4.1: configure
> ghc-boot-8.4.1: build
> process-1.6.3.0: configure
> ghc-boot-8.4.1: copy/register
> process-1.6.3.0: build
> ghci-8.4.1: configure
> ghci-8.4.1: build
> process-1.6.3.0: copy/register
> haskeline-0.7.4.2: configure
> Cabal-2.2.0.1: download
> haskeline-0.7.4.2: build
> Cabal-2.2.0.1: configure
> Cabal-2.2.0.1: build
> haskeline-0.7.4.2: copy/register
> ghci-8.4.1: copy/register
> Cabal-2.2.0.1: copy/register
> ghc-paths-0.1.0.9: configure
> ghc-paths-0.1.0.9: build
> ghc-paths-0.1.0.9: copy/register
> Building all executables for `intero' once. After a successful build of all of them, only specified executables will be rebuilt.
> intero-0.1.30: configure (exe)
> Configuring intero-0.1.30...
> Warning:
>     This package indirectly depends on multiple versions of the same package. This is very likely to cause a compile failure.
>       package time (time-1.8.0.2-4PdnJJ2JqJh910T92aaWLf) requires Win32-2.5.4.1-ChT90SipgKODMB4TIQfpw2
>       package process (process-1.6.3.0-EMLh1UiSoEGFNCguBqCrUe) requires Win32-2.5.4.1-ChT90SipgKODMB4TIQfpw2
>       package intero (intero-0.1.30) requires Win32-2.5.4.1-ChT90SipgKODMB4TIQfpw2
>       package haskeline (haskeline-0.7.4.2-J5dEXc5qDuD4rEVBJ8bNl0) requires Win32-2.5.4.1-ChT90SipgKODMB4TIQfpw2
>       package directory (directory-1.3.1.5-GNOmUyzbfS9o1zz7IobAH) requires Win32-2.5.4.1-ChT90SipgKODMB4TIQfpw2
>       package time (time-1.8.0.2) requires Win32-2.6.1.0
>       package process (process-1.6.3.0) requires Win32-2.6.1.0
>       package ghc (ghc-8.4.1) requires Win32-2.6.1.0
>       package directory (directory-1.3.1.5) requires Win32-2.6.1.0
>       package process (process-1.6.3.0) requires directory-1.3.1.5
>       package hpc (hpc-0.6.0.3) requires directory-1.3.1.5
>       package ghc-boot (ghc-boot-8.4.1) requires directory-1.3.1.5
>       package ghc (ghc-8.4.1) requires directory-1.3.1.5
>       package process (process-1.6.3.0-EMLh1UiSoEGFNCguBqCrUe) requires directory-1.3.1.5-GNOmUyzbfS9o1zz7IobAH
>       package intero (intero-0.1.30) requires directory-1.3.1.5-GNOmUyzbfS9o1zz7IobAH
>       package haskeline (haskeline-0.7.4.2-J5dEXc5qDuD4rEVBJ8bNl0) requires directory-1.3.1.5-GNOmUyzbfS9o1zz7IobAH
>       package ghc-boot (ghc-boot-8.4.1-IHWUCtBeBegEPdSkyRfjFK) requires directory-1.3.1.5-GNOmUyzbfS9o1zz7IobAH
>       package ghci (ghci-8.4.1) requires ghc-boot-8.4.1
>       package ghc (ghc-8.4.1) requires ghc-boot-8.4.1
>       package ghci (ghci-8.4.1-1pTOTAnCakPDhi0RhhV3qx) requires ghc-boot-8.4.1-IHWUCtBeBegEPdSkyRfjFK
>       package ghc (ghc-8.4.1) requires ghci-8.4.1
>       package intero (intero-0.1.30) requires ghci-8.4.1-1pTOTAnCakPDhi0RhhV3qx
>       package ghc (ghc-8.4.1) requires process-1.6.3.0
>       package intero (intero-0.1.30) requires process-1.6.3.0-EMLh1UiSoEGFNCguBqCrUe
>       package haskeline (haskeline-0.7.4.2-J5dEXc5qDuD4rEVBJ8bNl0) requires process-1.6.3.0-EMLh1UiSoEGFNCguBqCrUe
>       package hpc (hpc-0.6.0.3) requires time-1.8.0.2
>       package ghc (ghc-8.4.1) requires time-1.8.0.2
>       package directory (directory-1.3.1.5) requires time-1.8.0.2
>       package random (random-1.1-8TMFyS55oW94YjPcxq0zRd) requires time-1.8.0.2-4PdnJJ2JqJh910T92aaWLf
>       package intero (intero-0.1.30) requires time-1.8.0.2-4PdnJJ2JqJh910T92aaWLf
>       package directory (directory-1.3.1.5-GNOmUyzbfS9o1zz7IobAH) requires time-1.8.0.2-4PdnJJ2JqJh910T92aaWLf
> intero-0.1.30: build (exe)
> Preprocessing executable 'intero' for intero-0.1.30..
> Building executable 'intero' for intero-0.1.30..
> [ 1 of 10] Compiling Intero.Compat    ( src\Intero\Compat.hs, .stack-work\dist\22b940d3\build\intero\intero-tmp\Intero\Compat.o )
> [ 2 of 10] Compiling GhciTypes        ( src\GhciTypes.hs, .stack-work\dist\22b940d3\build\intero\intero-tmp\GhciTypes.o )
> 
> [ 3 of 10] Compiling GhciMonad        ( src\GhciMonad.hs, .stack-work\dist\22b940d3\build\intero\intero-tmp\GhciMonad.o )
> src\GhciMonad.hs:48:1: error:
>     Could not find module `GHCi.ObjLink'
>     Use -v to see a list of the files searched for.
>    |
> 48 | import GHCi.ObjLink as ObjLink
>    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 
> Completed 13 action(s).
> 
> --  While building custom Setup.hs for package intero-0.1.30 using:
>       C:\stack\setup-exe-cache\x86_64-windows\Cabal-simple_Z6RU0evB_2.2.0.0_ghc-8.4.1.exe --builddir=.stack-work\dist\22b940d3 build exe:intero --ghc-options " -ddump-hi -ddump-to-file -fdiagnostics-color=always"
>     Process exited with code: ExitFailure 1

**Comment by jmn** [comment]
Does AppVeyor succceed in building Intero 0.1.30 with Win64 GHC 8.4.1? It fails for me also.

**Comment by chrisdone** [comment]
I was waiting on this issue being fixed before AppVeyor would pass: https://github.com/haskell/network/issues/313

**Comment by k-bx** [comment]
FYI 8.4 works for me on older intero

**Comment by ntc2** [comment]
I assume this issue should be closed since most recent intero supports GHC 8.6.3 now, but I don't know anything about the AppVeyor situation.
